note.md

```
@@ -0,0 +1,100 @@
+**I. Standard E-commerce Purchase Flow**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)